datastorages/cmis/src/main/java/org/fao/geonet/api/records/attachments/CMISStore.java

```diff
@@ -391,6 +391,7 @@ public String delResources(final ServiceContext context, final int metadataId) t
             folderKey = getMetadataDir(context, metadataId);
             final Folder folder = cmisUtils.getFolderCache(folderKey, true);
 
+            Log.info(Geonet.RESOURCES, String.format("Deleting the folder of '%s' and the files within the folder", folderKey));
```

Comment:
This log entry should be added for all other file stores

Reply:
ok, i have added more logs and the logs are more generic in this case with metadataId.